packages/vow/src/watch-utils.js

```diff
@@ -72,13 +72,23 @@ export const prepareWatchUtils = (
     },
     {
       utils: {
-        /** @param {unknown[]} specimens */
+        /**
+         * @template {readonly unknown[]} TS
+         * @param {TS} specimens
+         * @returns {Vow<{ [P in keyof TS]: Fulfilled<TS[P]> }>}
```

Comment:
"T" is clutter. everything is a type here

```suggestion
         * @template {readonly unknown[]} S
         * @param {S} specimens
         * @returns {Vow<{ [P in keyof S]: Fulfilled<S[P]> }>}
```

same for other `TS`